lib/ansible/modules/async_wrapper.py

```diff
@@ -114,15 +114,15 @@ def _filter_non_json_lines(data):
     return ('\n'.join(lines), warnings)
 
 
-def _get_interpreter(module_path):
+def _get_interpreter(module_path: str) -> str:
```

Comment:
This is probably something like
```suggestion
def _get_interpreter(module_path: bytes | int | str | PathLike[bytes] | PathLike[str]) -> list[bytes]:
```